layouts/partner-form/single.html

```diff
@@ -0,0 +1,161 @@
+              nextButton.setAttribute("style", "display:none;");
+              submitButton.setAttribute("style", "display:inline-block;");
+            }
+          }
+        }
+      }
+    }
+
+    backButton.onclick = function (event) {
+      // no back button on first step
+      if(index == 1 && backButton) {
+        backButton.setAttribute("style", "display:none");
+      }
+      parent[index].setAttribute("style", "display:none;");
+      if(progressBar) {
+        progressBar.innerHTML = index
+        progressBarValue = progressBarValue - 8.333;
+        progressBar.setAttribute("style", `width: ${progressBarValue}%`);
+      }
+      index = index - 1
+      parent[index].setAttribute("style", "display:block;");
+    }
+
+    submitButton.onclick = function (event) {
+      var userInput = document.querySelectorAll(".user-input") // all non-selecter user inputs
+
+      for (i = 0; i < userInput.length; i++) {
+        user[userInput[i].name] = userInput[i].value
+      }
+
+      var useCaseOptions = document.getElementsByClassName('use-case');
+      if(useCaseOptions) {
+        user.use_case = [];
+        for (var i = 0, length = useCaseOptions.length; i < length; i++) {
+          if (useCaseOptions[i].selected) {
+            user.use_case.push(useCaseOptions[i].value);
+          }
+        }
+      }
+
+      var dataAmountOptions = document.getElementsByClassName('data-amount');
+
+      if(dataAmountOptions) {
+        for (var i = 0, length = dataAmountOptions.length; i < length; i++) {
+          if (dataAmountOptions[i].selected) {
+             user.data_amount = dataAmountOptions[i].value;
+             break;
+          }
+        }
+      }
+
+      var fileSizeOptions = document.getElementsByClassName('file-size');
+
+      if(fileSizeOptions) {
+        for (var i = 0, length = fileSizeOptions.length; i < length; i++) {
+          if (fileSizeOptions[i].selected) {
+           user.file_sizes = fileSizeOptions[i].value;
+           break;
+          }
+        }
+      }
+
+      var perMonthOptions = document.getElementsByClassName('per-month');
+
+      if(perMonthOptions) {
+        for (var i = 0, length = perMonthOptions.length; i < length; i++) {
+          if (perMonthOptions[i].selected) {
+             user.download_times_per_month = perMonthOptions[i].value;
+             break;
+           }
+        }
+      }
+
+      // timestamps with timezone
+      user.timestamp = Math.floor((new Date()).getTime() / 1000)
+
+      analytics.identify(analytics.user().anonymousId(),
+        {email: user.email, timestamp: user.timestamp, partner_form: true},
+        {integrations: {'All': false, 'Customer.io': true}}, function() {
+          analytics.track("partner-form", {});
+      });
+
+      analytics.identify(analytics.user().anonymousId(),
+        user,
+        {integrations: {'All': false, 'postgresSelfHosted': true}}, function() {
+          window.location = "/thank-you"
```

Comment:
On mobile I get a weird display once done filling out the form
![image](https://user-images.githubusercontent.com/7489221/71133045-62eb0780-21c7-11ea-8510-08329826a10c.png)